test/test-source-map-consumer.js

```diff
@@ -1134,3 +1134,18 @@ exports['test non-normalized sourceRoot (from issue #227)'] = function (assert)
   // Before the fix, this threw an exception.
   consumer.sourceContentFor(consumer.sources[0]);
 };
+
+exports['test webpack URL resolution'] = function (assert) {
+  var map = {
+    "version":3,
+    "sources": ["webpack:///webpack/bootstrap 67e184f9679733298d44"],
+    "names":[],
+    "mappings":"CAAS",
+    "file":"static/js/manifest.b7cf97680f7a50fa150f.js",
+    "sourceRoot":""
```

Comment:
nit: no need to quote the keys here :) Also, please add spaces after the colons.

(would make sense to add `prettier` eventually)